aws_lambda/sqs_handler.py

```diff
@@ -83,6 +83,7 @@ def _validate_candidate_set(candidate_set: Dict[str, Any]):
 
 def _validate_candidate(candidate: Dict[str, Any]):
     _validate_dict_value_type(candidate, 'item_id', int)
+    _validate_dict_value_type(candidate, 'url', str)
```

Comment:
This might address Jesh's comment:
```suggestion
    try:
        _validate_dict_value_type(candidate, 'url', str)
    except AssertionError:
        _validate_dict_value_type(candidate, 'item_id', int)
```
^ Untested